docs/root/configuration/listeners/listener_filters/proxy_protocol.rst

```diff
@@ -42,7 +42,7 @@ This filter emits the following general statistics, rooted at *downstream_proxy_
   not_found_disallowed, Counter, "Total number of connections that don't contain the PROXY protocol header and are rejected."
   not_found_allowed, Counter, "Total number of connections that don't contain the PROXY protocol header, but are allowed due to :ref:`allow_requests_without_proxy_protocol <envoy_v3_api_field_extensions.filters.listener.proxy_protocol.v3.ProxyProtocol.allow_requests_without_proxy_protocol>`."
 
-The filter also emits the statistics rooted at *downstream_proxy_proto.versions.<version>*
+The filter also emits the statistics rooted at *downstream_proxy_proto.<stat_prefix>.versions.<version>*
```

Comment:
`downstream_proxy_proto.[<stat_prefix>.]versions.<version>` ?

Reply:
Done, thanks. Also got rid of `downstream_` prefix, that is wrong and I changed it in my previous PR but forgot to update docs.